src/settings/SubjectSourcesSettings.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { ControlledVocab } from '@folio/stripes/smart-components';
+import { getSourceSuppressor } from '@folio/stripes/util';
+import {
+  IntlConsumer,
+  TitleManager,
+} from '@folio/stripes/core';
+
+import { RECORD_SOURCE } from '../constants';
+
+class SubjectSourcesSettings extends React.Component {
```

Comment:
Could you make the new components functional please?

Reply:
Hi @BogdanDenis Done